buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -0,0 +1,49 @@
+package org.elasticsearch.gradle.testclusters;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+
+public class RunTask extends DefaultTask {
+
+    private Boolean debug = false;
+
+    private ElasticsearchCluster cluster;
+
+    @Option(
+        option = "debug-jvm",
+        description = "Enable debugging configuration, to allow attaching a debugger to elasticsearch."
+    )
+    public void setDebug(boolean enabled) {
+        this.debug = debug;
+    }
+
+    @Input
+    public Boolean getDebug() {
+        return debug;
+    }
+
+    @Input
```

Comment:
If anything this should be `@Nested` but given this task isn't incremental (it produces no output and should always run) it would probably make more sense to make these `@Internal` so we don't bother wasting time on input snapshotting.